release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1285,8 +1285,14 @@ An overview of etcd, including the benefits it provides and how it works, is now
 
 [id="ocp-4-13-insights-operator"]
 === Insights Operator
+In OpenShift Container Platform 4.13, the Insights Operator now collects the following information:
 // https://issues.redhat.com/browse/OCPBUGS-6832
-The Insights Operator can now collect the `openshift_apps_deploymentconfigs_strategy_total` metric. This metric gathers deployment strategy information from a deployment's configuration.
+
+* The `openshift_apps_deploymentconfigs_strategy_total` metric. This metric gathers deployment strategy information from a deployment's configuration.
+
+* Additional machine resource definitions to identify why machines may be failing. 
+
+* The `ingresscontroller.operator.openshift.io` resource to inform Insights if the Authentication Operator is degraded.
```

Comment:
We should probably say `default` ingresscontroller.operator.openshift.io` resource.

Reply:
fixed